src/utility/errors.ts

```diff
@@ -1,10 +1,13 @@
 import { AxiosError } from 'axios';
+import { isAxiosError } from './axios';
 
-export function formatMessage(error: Error, message: string, defaultErrorMessage = 'Internal Server Error') {
+export function formatMessage(error: Error, message = '', defaultErrorMessage = 'Internal Server Error') {
   let errorMessage = defaultErrorMessage;
-  const axiosError = error as AxiosError;
-  if (axiosError?.response?.data?.errors && axiosError?.response.data.errors.length > 0) {
-    errorMessage = axiosError?.response.data.errors[0].message;
+  if (isAxiosError(error)) {
+    const axiosError = error as AxiosError;
```

Comment:
This cast shouldn't be necessary here since the type will already be inferred by the `isAxiosError` type guard.